README.md

```diff
@@ -90,7 +90,7 @@ you prefer to only receive updates for tagged releases, use
 
 ## Contributing
 
-See the [CONTRIBUTING.md][] document.
```

Comment:
im not sure I like this change as I would like to point out that the file name is part of the repo and is called CONTRIBUTING.md
 I'll commit this without this change. Thanks for pointing out the problem with the other link.